packages/dx-react-grid/src/plugins/table-selection.jsx

```diff
@@ -41,6 +45,9 @@ export class TableSelection extends React.PureComponent {
         {showSelectionColumn && (
           <Getter name="tableColumns" computed={tableColumnsComputed} />
         )}
+        {showSelectionColumn && (
```

Comment:
Maybe we should combine this both conditions to one?

Reply:
I've tried this initially but it seems that if 2 getters are rendered within React.Fragment then one of these getters stops working